crates/bevy-inspector-egui/src/reflect_inspector/mod.rs

```diff
@@ -894,7 +887,10 @@ impl InspectorUi<'_, '_> {
         id: egui::Id,
         options: &dyn Any,
     ) -> bool {
-        let type_info = value.get_type_info();
+        let Some(type_info) = value.get_represented_type_info() else {
```

Comment:
I panicked with a message similar to the `unreachable` call below, as this can only happen when you create a dynamic enum that does not represent any type, which AIUI should not be able to happen with a well-formed reflect impl.

Reply:
I am leaving it as is from https://github.com/jakobhellermann/bevy-inspector-egui/pull/137 as left by @Vrixyz

Reply:
I changed `get_type_info` to `get_represented_type_info` because I thought `get_type_info`  was removed as I had a compile error leading me that way, but I might have been mistaken as I see it in there: https://docs.rs/bevy/0.11.0/bevy/reflect/struct.TypeRegistryInternal.html#method.get_type_info ; maybe I was just missing a trait/derive or import ?

Reply:
@Vrixyz nope that part's correct, I just highlighted the wrong line... I'm referring to the error handling below